views/sections/SignIn/index.js

```diff
@@ -21,26 +27,90 @@ const SignIn = ({ progress, error, signIn }: Props) => {
     [signIn, email, password]
   )
 
+  useEffect(() => {
+    const _onInit = (auth2) => {
+      console.log('init OK', auth2)
+    }
+    const _onError = (err) => {
+      console.log('error', err)
+    }
+    window.gapi.load('auth2', () => {
+      window.gapi.auth2
+        .init({
+          client_id:
+            '172622657837-ac622rrsh2miq8din8h5muof9ki40927.apps.googleusercontent.com',
+        })
+        .then(_onInit, _onError)
+    })
+  })
+
+  const googleSignIn = (e) => {
+    // eslint-disable-next-line no-unused-expressions
+    e?.preventDefault()
+    const auth2 = window.gapi.auth2.getAuthInstance()
+    auth2.signIn().then((googleUser) => {
+      const profile = googleUser.getBasicProfile()
+      console.log(`ID: ${profile.getId()}`)
+      console.log(`Image URL: ${profile.getImageUrl()}`)
+      console.log(`Email: ${profile.getEmail()}`)
+      const { id_token } = googleUser.getAuthResponse()
+      console.log(`ID Token: ${id_token}`)
+    })
```

Comment:
Do we need auth through Google and Facebook for now? It also requires extra logic on the backend